README.md

```diff
@@ -23,9 +23,53 @@ Or download a prebuilt binary from [releases](./releases).
 ```bash
 protoc 
   --typescript-http_out [OUTPUT DIR] \
+  --typescript-http_opt use_enum_numbers=true,use_multi_line_comment=true
   [.proto files ...]
 ```
 
+#### Support options
+
+```ts
+// UseProtoNames controls the casing of generated field names.
+// If set to true, fields will use proto names (typically snake_case).
+// If omitted or set to false, fields will use JSON names (typically camelCase).
+use_proto_names: bool
+
+// UseEnumNumbers emits enum values as numbers.
+use_enum_numbers: bool
+
+// The method names of service methods naming case.
+// Only work when `UseEnumNumbers=true`
+// opt:
+// camelcase: convert name to lower camel case like `camelCase`
+// pascalcase: convert name to pascalcase like `PascalCase`
+// default is pascalcase
+enum_field_naming: string
+
+// Generate comments as multiline comments.
+// multiline comments: /** ... */
+// single line comments: // ...
+use_multi_line_comment: bool
```

Comment:
Why does the comment formatting need to be configurable?
I'd rather pick one of the formats (single/multi) and always use that, not configurable.

Reply:
because multiline comments is readable in ide tips dialog.
<img width="363" alt="image" src="https://github.com/einride/protoc-gen-typescript-http/assets/37169906/d3554a13-f9a3-4319-8d5b-4b810b4eae5c">

<img width="378" alt="image" src="https://github.com/einride/protoc-gen-typescript-http/assets/37169906/1c2fee6b-0829-4126-9480-ba1459cedfb0">

and make comment formatting be configurable is aviod break change(Maybe it's just me worrying too much).